app/src/components/email-input.tsx

```diff
@@ -14,21 +14,25 @@ export default function EmailInput({
   t,
   name = t("email-address"),
   id = "email",
+  disabled,
```

Comment:
Set a default value of `false` here, so you don't have to specify it if the input shouldn't be disabled.
```suggestion
  disabled = false,
```